src/js/background.js

```diff
@@ -765,6 +765,35 @@ Badger.prototype = {
     chrome.browserAction.setTitle({tabId: tab.id, title: "Privacy Badger"});
   },
 
+  debugSite: function() {
+    let self = this;
+    chrome.tabs.query({'active': true}, (tabs) => {
+      let tab = tabs[0];
+      console.log('using tab with url: ' + tab.url);
+      let browser = window.navigator.userAgent;
+      let version = chrome.runtime.getManifest().version;
+      let fqdn = window.extractHostFromURL(tab.url);
+      /* origin info:
+       * {'info': {'trackers':[{'origin': origin, 'action': action, 'snitch_map': snitch_map}, ...],
+       *           'browser': useragent,
+       *           'version': pbversion,
+       *           'fqdn': fqdn}}
+       */
+      let out = {info: {trackers: [], 'browser': browser, 'version': version, 'fqdn': fqdn}};
+      let action_map = self.storage.getBadgerStorageObject('action_map');
+      let snitch_map = self.storage.getBadgerStorageObject('snitch_map');
+      self.getAllOriginsForTab(tab.id).forEach((origin) => {
+        let base = window.getBaseDomain(origin);
+        out.info.trackers.push({
+          'origin': origin,
+          'baseDomain': base,
+          'action_info': action_map.hasItem(base) ? action_map.getItem(base) : 'no action info',
```

Comment:
I currently ask people to provide [action map entries for all related domains](https://github.com/EFForg/privacybadger/issues/1449#issuecomment-310113160) (via a simple `indexOf` string match). The information here is limited to the specific domain, which is less useful.